engines/engine.h

```diff
@@ -235,6 +235,10 @@ class Engine {
 	GUI::Debugger *_debugger;
 public:
 
+	enum VideoMode {
+		kCGAMode,
+		kHercMode,
+	};
```

Comment:
We already have an enum for these, RenderMode. Check [common/rendermode.h](https://github.com/scummvm/scummvm/blob/master/common/rendermode.h)